packages/overlay/stories/overlay.stories.ts

```diff
@@ -637,3 +644,51 @@ export const superComplexModal = (): TemplateResult => {
         </overlay-trigger>
     `;
 };
+
+export const virtualElement = (args: Properties): TemplateResult => {
+    const pointerenter = async (event: PointerEvent): Promise<void> => {
+        event.preventDefault();
+        const trigger = event.target as HTMLElement;
+        const virtualTrigger = new VirtualTrigger(event.clientX, event.clientY);
+        openOverlay(
+            trigger,
+            'modal',
+            trigger.nextElementSibling as HTMLElement,
```

Comment:
If I'm understanding this right, clients still pass in an HTMLElement, but now that's just for event bubbling (and focus?), but the virtualTrigger manages positioning. Seems good to me!